templates/base.jinja2

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<title>Python Learning Journal</title>
+		<link rel="stylesheet" type="text/css" href="/static/stylesheets/style.css">
+	</head>
+
+	<body>
+	<img id="bg" src="/static/img/Python-logo-notext.svg">
+	<div id="container">
+		<header id="header">
+			<h1>Python Learning Journal</h1>
+
+			<aside id="user-controls">
+                {% if not request.authenticated_userid %}
+                    <button id="button-forward" onclick="window.location='{{ request.route_url('login') }}'">
+                        Login
+                    </button>
```

Comment:
I'd find a different place to put this login button.  It looks awkward where it is now.